### PR TITLE
fine: content distribution

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Provisional PR for the initial release of content distribution. 

NB:
only solid pill up to date

### CODE REVIEW

File: pkg/arvo/sys/vane/ames.hoon [hunk 1]
```diff
@@ -2307,6 +2699,658 @@
...
+              (ke-resend [fra hoot]:u.want)
+            --
+          --
+        ::
+        ++  on-pine-plea
+          |=  [=ship payload=*]
+          ^+  event-core
+          =+  ;;(blk=balk payload)
+          ?>  =(%c van.blk)
+          =.  car.blk  %w
```
Comment (tacryt-socryp) [inline pkg/arvo/sys/vane/ames.hoon:3111]: `=:` seems best for these

File: pkg/arvo/sys/vane/ames.hoon [hunk 2]
```diff
@@ -485,6 +553,7 @@
 ::    0b11: comet          -- 16 bytes
 ::
 +$  rank  ?(%0b0 %0b1 %0b10 %0b11)
++$  byuts  [wid=@ud dat=@ux]
```
Comment (belisarius222) [inline pkg/arvo/sys/vane/ames.hoon:556]: unused

File: pkg/arvo/sys/vane/ames.hoon [hunk 3]
```diff
@@ -622,6 +811,9 @@
       $%  [%wait date=@da]
           [%rest date=@da]
       ==  ==
+      $:  %c
```
Comment (belisarius222) [inline pkg/arvo/sys/vane/ames.hoon:814]: I think this is unused too, now that %bide has been removed

**Comment by zalberico** [comment]
Superseded by #5725, closing

**Comment by jalehman** [comment]
This is closed because it's superseded by itself?

**Comment by zalberico** [comment]
Edited the comment with the proper version #

**Pull request closed without merging**